index.html

```diff
@@ -27,36 +27,37 @@ <h2></h2>
     <div class="container">
       <section id="main_content">
         <h3>
-<a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span class="octicon octicon-link"></span></a>Welcome to my GitHub Pages.</h3>
+<a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true">
+    <span class="octicon octicon-link"></span></a>Welcome to my GitHub Pages.</h3>
 
-<p>Here is my Github generated page. but first, let me introduce myself.</p>
-<p>My name is Chokri. I'm a webdesigner and developper.</p>
-<p>The Web is a passion. Followed by a graduate studies in Multimedia & technology it gaves me the opportunity to make rich internet applications based on design patterns like MVC and to boost my productivity by using CakePHP as Framework's JQuery.
-Beyond web development, I am also interested in the graphic side of web interfaces to make use of GUI more attractive and more ergonomic.</p>
-<p>If you're using the GitHub for Mac, simply sync your repository and you'll see the new branch.</p>
+<p>My name is Chokri. I become a R&D Consulting after 8 years of contributions on a big local projects.</p>
+<p>I'm mastering PHP Frameworks & ES6 JavaScript (Node JS, React, Angular)</p>
+<p>The Web is my passion. Followed by a graduate studies in Multimedia & computer sciences and technologies.
+Beyond web development, I am also interested in Linux and Open Source softwares.</p>
+<p>I contribute to Symfony Project, Drupal project and CakePHP Framework</p>
+<p>I write posts about my experiences, and my new tech discoveries in my blog [<a href="https://khalifa.tn">C. Khalifa</a>]</p>
 
 <h3><a id="designer-templates" class="anchor" href="#designer-templates" aria-hidden="true"><span class="octicon octicon-link"></span></a>You want to follow me?</h3>
 
 <p>I use many social website like many Internet users, but you will find me on other social networks maybe you know them and maybe not like:</p>
 <ul>
-  <ol><a href="http://facebook.com/chokrikhalifa">Facebook</a></ol>
-  <ol><a href="http://twitter.com/shuck_">twitter</a></ol>
-  <ol><a href="https://koding.com/chokri">Koding</a></ol>
+  <li><a href="http://twitter.com/shuck_">🖥 @shuck_</a></li>
```

Comment:
No need to the emoji here
```suggestion
  <li><a href="http://twitter.com/shuck_"> @shuck_</a></li>
```